helfi_platform_config.libraries.yml

```diff
@@ -145,14 +145,23 @@ watson_talpa:
   js:
     'https://coh-chat-app-prod.ow6i4n9pdzm.eu-de.codeengine.appdomain.cloud/widget.min.js': {
       type: external,
+      attributes: {
```

Comment:
Should these async parameters be added to all Watson libraries, since they're pretty much using the same code?

Reply:
Fixed 👍